code/__defines/modular_guns.dm

```diff
@@ -2,6 +2,8 @@
 #define CHASSIS_MEDIUM 3
 #define CHASSIS_LARGE 4
 
+#define NO_PIN "nopin"
```

Comment:
this seems a lot more unintuitive than just making null pin not have a pin, and then adding a /obj/item/pin/random as the default